config/settings/common_settings.py

```diff
@@ -94,11 +94,19 @@ def pipe_delim(pipe_string):
     db_key: db_url(url_string)
     for db_key, url_string in decouple_config('MATHESAR_DATABASES', cast=Csv(pipe_delim))
 }
-DATABASES[decouple_config('DJANGO_DATABASE_KEY', default="default")] = decouple_config('DJANGO_DATABASE_URL', cast=db_url)
+
+try:
+    DATABASES[decouple_config('DJANGO_DATABASE_KEY', default="default")] = decouple_config('DJANGO_DATABASE_URL', cast=db_url)
```

Comment:
Wouldn't it be better to set the default SQLite dictionary as a default value? The default dictionary can be stored in a variable and used as a default value 
```suggestion
DEFAULT_DATABASE = {
        'ENGINE': 'django.db.backends.sqlite3',
        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
    }
DATABASES[decouple_config('DJANGO_DATABASE_KEY', default="default")] = decouple_config('DJANGO_DATABASE_URL', cast=db_url, default=DEFAULT_DATABASE
```

Reply:
Wow, really clean, but this doesn't work because of `cast`. I've added the connection string as the `default` now and it works flawlessly.